tests/openadapt/test_share.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import unittest
+import os
+from unittest.mock import patch
+from openadapt import share
+
+
+class ShareTestCase(unittest.TestCase):
+    def test_export_recording_to_folder(self):
```

Comment:
Please add a newline before function definition 🙏

Reply:
addressed in https://github.com/MLDSAI/OpenAdapt/pull/223/commits/3434a00b4134bd756043123a23458b3e1c4126ed